src/RentVehicle.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Date;
+
+public class RentVehicle {
+    public static Rent rentVehicle(Vehicle vehicle, double cost,  MechanicService mechanicService)
+            throws DefectedVehicleException {
+        if (!mechanicService.isBroken(vehicle)) {
+            return new Rent(vehicle.getId(), new Date(), cost);
+        }
+
+        throw new DefectedVehicleException("Vehicle is defected");
+    }
```

Comment:
public static Rent rentVehicle(Vehicle vehicle, double cost,  MechanicService mechanicService)
            throws DefectedVehicleException {
        if (mechanicService.isBroken(vehicle)) {
              throw new DefectedVehicleException("Vehicle is defected");
        }
        return new Rent(vehicle.getId(), new Date(), cost);
    }

such way looks better)

Reply:
Thanks, it's really better, fixed